flatpak/com.asimov_platform.asimov_cli.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: com.asimov_platform.asimov_cli
```

Comment:
@AlexSanches1 Let's change the app ID to `so.asimov.cli`.

Reply:
@artob it is done